otter/tap/api.py

```diff
@@ -317,11 +318,10 @@ def setup_converger(parent, kz_client, dispatcher, interval, build_timeout,
         kz_client=kz_client,
         interval=interval,
         partitioner_path=CONVERGENCE_PARTITIONER_PATH,
-        time_boundary=15,  # time boundary
+        time_boundary=1,  # time boundary
```

Comment:
Oh good catch. That was not meant to be committed. I was just testing it. Will change it back. It is the time partitioner waits before it allocates the buckets. Say if it is 15 seconds and no nodes must join or leave the partition for 15 seconds before it says allocated.